packages/evolution-common/package.json

```diff
@@ -24,7 +24,8 @@
     "chaire-lib-common": "^0.2.2",
     "i18next": "^22.4.15",
     "lodash": "^4.17.21",
-    "typescript": "^4.9.4"
+    "typescript": "^4.9.4",
+    "uuid": "^9.0.1"
```

Comment:
This adds a dependency to yet another uuid package. Please use the same as other packages, or update all of them everywhere (including transition, you might prefer just using the same as elsewhere)

Reply:
You mean you want me to update all uuid versions in other packages to 9.0.1? Wouldn't it be better to wait for current surveys to be completed?

Reply:
I normalized all uuid versions to 8.3.2